### ISSUE TITLE
How to use a text prompt.

### ISSUE THREAD

**Issue by GitChanyoung** [yformer/EfficientSAM#26]
Thank you for your amazing research. I would like to use a text prompt. However, when I examined the code, there was no code related to the text prompt. I'm curious if you have any plans to add it in the future or if there's a specific reason why text prompts are not available. Thank you.

**Comment by yformer** [comment]
@GitChanyoung, thanks for your interest in EfficientSAM!  You can check Grounded EfficientSAM demo on [Grounded-Efficient-Segment-Anything](https://github.com/IDEA-Research/Grounded-Segment-Anything/tree/main/EfficientSAM), which you can perform text prompt. 

![image](https://github.com/yformer/EfficientSAM/assets/89545561/0a03fdea-304c-4b50-bfa7-0fb392a9cd0c)

**Comment by yformer** [comment]
@xiaobanni, Yes, Grounded-SAM is a two-stage process. We will consider the text prompt direction. Thanks for your interest.
